### ISSUE TITLE
Does not support inspecting Errors in v8.x

### ISSUE THREAD

**Issue by joyeecheung** [nodejs/llnode#143]
Looks like v8 has made some significant changes to errors (rewrite in C++), now they are no longer normal objects. The inspection seems to have stopped working since 7.x. I am working on a fix.

**Comment by mmarchini** [comment]
@joyeecheung is this still an issue?

**Comment by joyeecheung** [comment]
@mmarchini Yes.

With this:

```diff
diff --git a/test/fixtures/inspect-scenario.js b/test/fixtures/inspect-scenario.js
index 11133ac..03a508a 100644
--- a/test/fixtures/inspect-scenario.js
+++ b/test/fixtures/inspect-scenario.js
@@ -57,6 +57,10 @@ function closure() {
   );
   c.hashmap['buffer'] = Buffer.from([0xff, 0xf0, 0x80, 0x0f, 0x01, 0x00]);

+  c.hashmap['error'] = new Error('test');
+  c.hashmap['error'].code = 'ERR_TEST';
+  c.hashmap['error'].errno = 1;
+
   c.hashmap[0] = null;
   c.hashmap[4] = undefined;
   c.hashmap[23] = /regexp/;
```

I got:

```
.error=0x00000c0ad02d0b59:<unknown>,
```

**Comment by mmarchini** [comment]
@joyeecheung are you still working on this? If not, would you mind if I take this on?

**Comment by joyeecheung** [comment]
@mmarchini I probably lost the initial investigation with my last laptop...feel free to take this!

**Comment by misterdjules** [comment]
@mmarchini Isn't the `stack` property [an accessor that replaces itself with a data property at runtime with the formatted stack trace](https://github.com/v8/v8/blob/8109d7e1f11f42af08bf5cbe5ca1313f14589a55/src/accessors.cc#L1142-L1201)? Could that explain why llnode has some difficulties displaying some useful info about it?

**Comment by mmarchini** [comment]
@misterdjules yep, that's what I got so far as well. The actual stack trace seems to be stored in the `.<non-string>=0x00003347c1720bd1:<Array: length=46>`, unfolding it should be enough to get a nice stack trace for the Error object. I'm also working on https://github.com/nodejs/llnode/issues/156 to display the symbol of this property correctly.

**Comment by mmarchini** [comment]
Fixed by #215
